common/test/acceptance/tests/discussion/helpers.py

```diff
@@ -79,8 +79,13 @@ def enable_cohorting(self, course_fixture):
         enables cohorting for the current course fixture.
```

Comment:
Please update the doc so that it explains that it also sets always_divide_inline_discussions to True.